db/seeds.rb

```diff
@@ -55,6 +56,24 @@
   avatar: File.open("#{Rails.root}/db/seed_data/avatar3.png"),
   favourite_sports: 'Hardcore-Couching').save(validate: false)
 
+sports = %w(Fußball Basketball Tennis)
+
+6.times do
+  sports.shuffle
+
+  User.new(
+    first_name: Faker::Name.unique.first_name,
+    last_name: Faker::Name.unique.last_name,
+    email: Faker::Internet.unique.email,
+    password: 'seed1234',
+    password_confirmation: 'seed1234',
+    birthday: Faker::Date.birthday(18, 65),
+    telegram_username: 'eb',
+    telephone_number: '12345654321',
+    avatar: File.open("#{Rails.root}/db/seed_data/avatar3.png"),
+    favourite_sports: sports[rand(sports.length), rand(sports.length - 1) + 1].join(', ')).save(validate: false)
+end
```

Comment:
It's not really the job of the seeds file to add example data. The seeds file should only specify the minimal data that is needed for operating the system at all.

But honestly I think for now this is fine. Just consider opening a ticket to extract this behaviour into a sperate rake task. We also have another ticket for even more example data that you could link to: #224

Reply:
Jep, you are right. I added a comment to the ticket that you mentioned.